.gitignore

```diff
@@ -10,3 +10,6 @@ RUNNING_PID
 .secrets
 node_modules
 bundle.js
+.venv/
```

Comment:
Do these need to be added to our gitignore? We aren't using python.

Reply:
Removed it